packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -39,9 +39,9 @@ export class CoinbaseWalletProvider extends ProviderEventEmitter implements Prov
       if (!this.signer) {
         switch (args.method) {
           case 'eth_requestAccounts': {
-            const signerType = await this.requestSignerSelection();
-            const signer = this.initSigner(signerType);
-            await signer.handshake();
+            const signerType = await this.requestSignerSelection(args);
+            const signer = await this.initSigner(signerType);
```

Comment:
why await?

Reply:
wow you're fast. good catch!

removed [b490a3d](https://github.com/coinbase/coinbase-wallet-sdk/pull/1392/commits/b490a3ddd01002c46a0a0719f1bfdc7fe0341379)